allure-selenide/src/main/java/io/qameta/allure/selenide/AllureSelenide.java

```diff
@@ -91,10 +95,24 @@ public AllureSelenide disableLogs(final LogType logType) {
     }
 
     private static Optional<byte[]> getScreenshotBytes() {
+        return ScreenShotLaboratory.getInstance().getLastThreadScreenshot()
```

Comment:
what if you are using `saveScreenshots=true` and make a single screenshot manually at test start? It seems like you'll get the same screenshot in all the steps

Reply:
If I understand correctly, it should not be a case because Selenide will take a separate screenshot on each fail

Reply:
@dstekanov and what if we got a case, when someone manually produces a screenshot on fail to attach it to the Main Test Result Page and not under the steps, while also wanting that screenshot to be under given step? Also what if we use Junit assertAll with selenide boolean methods?